runtime/src/main/cpp/ObjCExport.mm

```diff
@@ -315,6 +315,8 @@ - (instancetype)copyWithZone:(NSZone *)zone {
 __attribute__((weak)) const ObjCTypeAdapter** Kotlin_ObjCExport_sortedProtocolAdapters = nullptr;
 __attribute__((weak)) int Kotlin_ObjCExport_sortedProtocolAdaptersNum = 0;
 
+__attribute__((weak)) bool Kotlin_ObjCExport_initTypeAdapters = false;
```

Comment:
Not very trivial state management in Kotlin_ObjCExport_initTypeAdapters. Could it be const, as only seems to be changed during linkage, and not runtime?

Reply:
Actually it is changed in runtime, handled with https://github.com/JetBrains/kotlin-native/pull/3440/files#diff-70fa52f175b3e9befea519e771fa08e2R440